tests/non-js-resources/subtests/binary/subtests/url/index.md

```diff
@@ -18,6 +18,10 @@ import binURL from './binary.bin';
 fetch(binURL).then(async r => {
   console.log(await r.arraybuffer());
 });
+
+// OR:
+const url = new URL('./binary.bin', import.meta.url);
+fetch(url).then(async r => console.log(await r.arraybuffer()));
 ```
 
```

Comment:
````suggestion
});
```

Bundlers may use something other than a static import to achieve this. The test passes as long as the resulting URL can be stored in a variable.

````

Reply:
This allows for `require()` too